src/prasad/FebonacciSeries.java

```diff
@@ -0,0 +1,37 @@
+package prasad;
+import java.util.Scanner;
+
+public class FebonacciSeries {
+	
+	void feboSeries(int n) {
+		int temp1=0;
+		int temp2=1;
+		int ans=0;
+		int count=0;
+		
+		int max=n-2;
+		while(count<=max) {
+			if(count==0) {
+				System.out.print(temp1+" "+temp2+ " ");
+				count++;
```

Comment:
As you increasing value of cnt in if & else condition both places so write after the condition
just to correct you